### ISSUE TITLE
YOLOv4 Training loss problem

### ISSUE THREAD

**Issue by Damon2019** [AlexeyAB/darknet#7434]
![图片](https://user-images.githubusercontent.com/42839230/109249877-84ac3980-7823-11eb-8da6-fd42580eef75.png)

I train my own dataset, having this problem training Yolov4，but Yolov3, Yolov3-tiny and Yolov4-tiny were all successful

**Comment by Damon2019** [comment]
I tested the model trained by Yolov4 and could not detect the target of the image

**Comment by versavel** [comment]
Did you use same learning rate ? Did you try lowering the training rate when training the Yolov4 model ? 

What about the other configuration parameters, did you use same values where to configuration parameters “overlap" ?

**Comment by Damon2019** [comment]
I have tried many times to modify the hyper-parameter and finally solved it. Thank you very much.

**Comment by lla-lla** [comment]
My situation is similar to yours. How did you solve it?

**Comment by lla-lla** [comment]
@Damon2019     My situation is similar to yours. How did you solve it?

**Comment by Damon2019** [comment]
> @Damon2019 My situation is similar to yours. How did you solve it?

lowering the training rate or momentum

**Comment by Damon2019** [comment]
> @Damon2019 My situation is similar to yours. How did you solve it?

lowering the training rate or momentum

**Comment by lla-lla** [comment]
@Damon2019  I brought the learning rate down to 1e-6 and it still didn't work,I'll try to adjust it momentum. thanks
